lib/web/apiserver.go

```diff
@@ -2865,6 +2868,43 @@ func (h *Handler) getClusters(w http.ResponseWriter, r *http.Request, p httprout
 	return out, nil
 }
 
+type GetClusterInfoResponse struct {
+	ui.Cluster
+	IsCloud bool `json:"isCloud"`
+}
+
+// getClusterInfo returns the information about the cluster in the :site param
+//
+// GET /v1/webapi/sites/:site/info
+//
+// Successful response:
+//
+//	{"name": "localhost","lastConnected": "RFC3339 time","status": "online","publicURL": "localhost:3080","authVersion": "17.0.0-dev","proxyVersion": "17.0.0-dev", "isCloud": true}
+func (h *Handler) getClusterInfo(w http.ResponseWriter, r *http.Request, p httprouter.Params, sctx *SessionContext, site reversetunnelclient.RemoteSite) (interface{}, error) {
+	ctx := r.Context()
+	c, err := ui.GetClusterDetails(ctx, site)
```

Comment:
nit: `c` reads fine now but I would spell it out as `clusterDetails` because it will stand out between `ctx`, `c`, `clt` etc.

Reply:
Agreed, thanks.